client/stylesheets/_responsive.scss

```diff
@@ -259,10 +299,13 @@
     .helpImgUl > li:nth-child(2) {
     	display: none !important;
     }
-    .profileCardLi {
-    	width: 90% !important;
-    }
-    .searchResultsCardLi {
-    	width: 90% !important;
+    .oneDiv {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `oneDiv` should be written in lowercase with hyphens